From b279faf5539677ced877945a9c16fa16e47606ed Mon Sep 17 00:00:00 2001
From: Min He <min.he@intel.com>
Date: Thu, 23 Mar 2017 04:44:43 +0800
Subject: [PATCH 0922/2367] drm/i915/gvt: Disable plane fault error interrupts

In Xen Dom0, we always see the error interrupt from PIPE, so disable
by not masking them as a workaround.

Change-Id: Ida5fe3037e3d547a066b460a814a98283435ef5a
Signed-off-by: Min He <min.he@intel.com>
Reviewed-on: https://android.intel.com/613345
Reviewed-by: Jiang, Fei <fei.jiang@intel.com>
Reviewed-by: Dong, Eddie <eddie.dong@intel.com>
Tested-by: Dong, Eddie <eddie.dong@intel.com>
---
 drivers/gpu/drm/i915/i915_irq.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index 5d34d48..506b445 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -3459,8 +3459,7 @@ static void gen8_de_irq_postinstall(struct drm_i915_private *dev_priv)
 	enum pipe pipe;
 
 	if (INTEL_GEN(dev_priv) >= 9) {
-		de_pipe_masked |= GEN9_PIPE_PLANE1_FLIP_DONE |
-				  GEN9_DE_PIPE_IRQ_FAULT_ERRORS;
+		de_pipe_masked |= GEN9_PIPE_PLANE1_FLIP_DONE;
 		de_port_masked |= GEN9_AUX_CHANNEL_B | GEN9_AUX_CHANNEL_C |
 				  GEN9_AUX_CHANNEL_D;
 		if (IS_GEN9_LP(dev_priv))
-- 
2.7.4

